phpcs.xml.dist

```diff
@@ -153,4 +153,15 @@
 			</property>
 		</properties>
 	</rule>
+	<rule ref="Gutenberg.CodeAnalysis.ForbiddenFunctionsAndClasses">
+		<include-pattern>./packages/block-library/src/*/*.php</include-pattern>
+		<properties>
+			<property name="forbidden_functions" type="array">
+				<element value="(G|g)utenberg.*"/>
+			</property>
+			<property name="forbidden_classes" type="array">
+				<element value="(G|g)utenberg.*"/>
```

Comment:
Nit: technically speaking, it should be enough to use the following:
```suggestion
				<element value="gutenberg.*"/>
			</property>
			<property name="forbidden_classes" type="array">
				<element value="Gutenberg.*"/>
```

Although, maybe I'm missing some special cases.

Reply:
You are right.
However, to be on the safer side, I would prefer to allow the first letter to be either 'G' or 'g'.
it's just my personal preference.